src/patterns/dosdp-patterns/cellPartOfAnatomicalEntity.yaml

```diff
@@ -0,0 +1,38 @@
+pattern_name: cellPartOfAnatomicalEntity
+pattern_iri: http://purl.obolibrary.org/obo/cl/cellPartOfAnatomicalEntity
+
+description: A cell that is part of an anatomical entity.
+examples: CL_0002319 neural cell, CL_0009001 compound eye retinal cell, CL_2000004 pituitary gland cell (654 total)
```

Comment:
neural cell is a bad example

animal cell and part of some nervous system

Is this right? Make a separate ticket for this. This would classify endothelial cells in brain blood vessels as neurons.